packages/lesswrong/lib/cookies/cookies.ts

```diff
@@ -104,6 +104,12 @@ export const HIDE_MORE_FROM_THE_FORUM_RECOMMENDATIONS_COOKIE = registerCookie({
   description: "Don't show the \"More from the forum\" recommendations section",
 });
 
+export const HIDE_NEW_POST_HOW_TO_GUIDE_COOKIE = registerCookie({
+  name: "hide_new_post_how_to_guide",
+  type: "functional",
```

Comment:
I thought we were making these type of things necessary.

Reply:
Good point - I think this code predates that decision